x/sequencer/keeper/rotation.go

```diff
@@ -0,0 +1,122 @@
+package keeper
+
+import (
+	"sort"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dymensionxyz/dymension/v3/x/sequencer/types"
+)
+
+func (k Keeper) startNoticePeriodForSequencer(ctx sdk.Context, seq *types.Sequencer) time.Time {
+	completionTime := ctx.BlockHeader().Time.Add(k.NoticePeriod(ctx))
+	seq.UnbondTime = completionTime
```

Comment:
Unbond time seems overloaded because this is not unbond time but actually the time of finishing the notice period

Reply:
u right but I didn't want to add new field.
If seq bonded: UnbondTime is notice period completion time
if seq unbonding: UnbondTime is unbonding completion time
want to suggest better name?

Reply:
I'd 100% just add a new field tbh
if you really want one then 'transitionTime' or just 'noticeOrUnbondTime'